cdap-hbase-compat-1.0/src/main/java/co/cask/cdap/data2/transaction/messaging/coprocessor/hbase10/MessageTableRegionObserver.java

```diff
@@ -143,12 +142,12 @@ private TopicMetadataCache getTopicMetadataCache(RegionCoprocessorEnvironment en
   }
 
   private TopicMetadataCache createTopicMetadataCache(RegionCoprocessorEnvironment env) {
-    return new TopicMetadataCache(env, cConfReader, nameConverter, hbaseNamespacePrefix, metadataTableNamespace,
+    return new TopicMetadataCache(env, cConfReader, hbaseNamespacePrefix, metadataTableNamespace,
                                   new HBase10ScanBuilder());
   }
 
   private Supplier<TransactionStateCache> getTransactionStateCacheSupplier(String tablePrefix, Configuration conf) {
-    String sysConfigTablePrefix = nameConverter.getSysConfigTablePrefix(tablePrefix);
+    String sysConfigTablePrefix = HTableNameConverter.getSysConfigTablePrefix(tablePrefix);
```

Comment:
Yes it is unrelated but I thought it would be good to make that change to use the static method.